Source/WebKit/UIProcess/Automation/WebAutomationSession.h

```diff
@@ -111,6 +111,8 @@ class AutomationCommandError {
 
 using AutomationCompletionHandler = WTF::CompletionHandler<void(std::optional<AutomationCommandError>)>;
 
+using ContextHandlePair = std::pair<Inspector::Protocol::Automation::BrowsingContextHandle, Inspector::Protocol::Automation::FrameHandle>;
```

Comment:
Nit: `PageAndFrameHandle` or `BrowsingContextHandlePair`.

just `context` by itself could refer to browsing context or user context.